plugins/request.rb

```diff
@@ -83,54 +127,27 @@ def parse_args(args)
         when '-o'
           options[:output_file] = File.expand_path(val)
         when '-u'
-          val = val.partition(':')
-          options[:auth_username] = val[0]
-          options[:auth_password] = val[2]
+          options[:auth_username] = val
+        when '-p'
```

Comment:
This -p option is inconsistent with curl which uses it as the short version of ```--proxytunnel```. I'd really prefer to leave the authentication with the -u option to emulate the behavior. Also -p doesn't appear in ```parse_args_http```.